### ISSUE TITLE
Minsky system dynamics software and agriculture

### ISSUE THREAD

**Issue by turinval** [ouisharelabs/food-dashboard#13]
HOw would one integrate an agri dashboard with an accounting system? And how would one integrate that into an economic model. Minsky system dynamics is based on Steve Keen's Minsky software. An open source tool to provide extensive collaboration in the agriculture exchanges. 

The dependence on subsidies for example is an example of where economic models can help farms figure out optimal decisions for sustainability. How to collaborate small farms and larger farms with urban markets.?

THe software we are doing is based on solving those questions using very simple system dynamics and business dashboards. I was wondering if anyone was interested to discuss this? I am trying to get a case study with farms and other developers to form a experiment to see computer aided analysis in economic decision making and participation increases.

**Comment by bhaugen** [comment]
In the [Value Flows](https://valueflo.ws/) we are working on an economic model and vocabulary that (we think) can handle agriculture as well as all other aspects of economic relationships. Our work is roughly based on an ISO standard economic and accounting ontology. We've had some discussions about how to generate the typical accounting reports from it. For example:
https://github.com/valueflows/valueflows/issues/141
https://github.com/valueflows/valueflows/issues/142

We haven't tried Keen's Minsky system yet, but if you go to work on an ag model there, I'd be very interested in going with you.

> THe software we are doing is based on solving those questions using very simple system dynamics and business dashboards.

Do you mean Minsky? Or some other software?

**Comment by turinval** [comment]
Yes I am very interested in agri modelling! Our main problem is a lack of being able to track down the dynamics and thus make the correct equations.

**Comment by turinval** [comment]
[PAPERMinskyDynamicIndicators.docx](https://github.com/ouisharelabs/food-dashboard/files/785403/PAPERMinskyDynamicIndicators.docx)

**Comment by turinval** [comment]
I feel a lot of the economics stuff in Minsky would be a huge catalysis for these projects of value accounting and ontology systems.

**Comment by bhaugen** [comment]
Here's something I wrote in 2005, before a lot of the projects we're working on now. It might fit into Minsky. 
http://mikorizal.org/BeyondTheEnterprise.html
Or not, I don't really understand Minsky yet, but like a lot of the glimpses I've seen. Which is why I'd like to follow along if you do something with agriculture or value accounting and Minsky. And I mean follow. I might be able to get some data, [for example, Nova Scotia fisheries](http://locecon.org/clusters/network/11/qty/fn/), but I got nothin about farm subsidies. So I'd just follow what you did and try to learn from it.
